.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,21 @@
+---
+name: "\U0001F41B Bug report"
+about: Report a bug about the Scala compiler or the standard library
+title: ''
+assignees: ''
+
+---
+
+## minimized code
```

Comment:
I worry that this might be too discouraging. Of course in the best case, issues are reported with a minimized, self-contained example, but we don't want to prevent people from reporting a bug just because they didn't manage to track it down to a small or deterministic example on their own. A bug that's not reported could be a lot more damaging than having to deal with out-of-scope issues here and there.

Reply:
For example, https://github.com/scala/bug/issues/11892 is a report that I consider appropriate. Maybe we won't figure out what the issue is and end up closing it unresolved, but it is still useful to build awareness, have a searchable reference, etc.

Reply:
perhaps “reproduction steps”, with a note “(preferably minimized)” in the body? or is even that too demanding?

Reply:
I took "minimized code" from Dotty, but my preference is calling this "steps".

We can also put a comment on top clarifying that none of the sections are mandatory.

Reply:
Soon I need to have the "reproduction steps" talk with my child.